lib/ansible/modules/network/fortimanager/fmgr_fwpol_ipv4.py

```diff
@@ -0,0 +1,1510 @@
+    choices: ["disable", "enable"]
+
+  captive_portal_exempt:
+    description:
+      - Enable to exempt some users from the captive portal.
+      - choice | disable | Disable exemption of captive portal.
+      - choice | enable | Enable exemption of captive portal.
+    required: false
+    choices: ["disable", "enable"]
+
+  block_notification:
+    description:
+      - Enable/disable block notification.
+      - choice | disable | Disable setting.
+      - choice | enable | Enable setting.
+    required: false
+    choices: ["disable", "enable"]
+
+  av_profile:
+    description:
+      - Name of an existing Antivirus profile.
+    required: false
+
+  auto_asic_offload:
+    description:
+      - Enable/disable offloading security profile processing to CP processors.
+      - choice | disable | Disable ASIC offloading.
+      - choice | enable | Enable auto ASIC offloading.
+    required: false
+    choices: ["disable", "enable"]
+
+  auth_redirect_addr:
+    description:
+      - HTTP-to-HTTPS redirect address for firewall authentication.
+    required: false
+
+  auth_path:
+    description:
+      - Enable/disable authentication-based routing.
+      - choice | disable | Disable authentication-based routing.
+      - choice | enable | Enable authentication-based routing.
+    required: false
+    choices: ["disable", "enable"]
+
+  auth_cert:
+    description:
+      - HTTPS server certificate for policy authentication.
+    required: false
+
+  application_list:
+    description:
+      - Name of an existing Application list.
+    required: false
+
+  application:
+    description:
+      - Application ID list.
+    required: false
+
+  app_group:
+    description:
+      - Application group names.
+    required: false
+
+  app_category:
+    description:
+      - Application category ID list.
+    required: false
+
+  action:
+    description:
+      - Policy action (allow/deny/ipsec).
+      - choice | deny | Blocks sessions that match the firewall policy.
+      - choice | accept | Allows session that match the firewall policy.
+      - choice | ipsec | Firewall policy becomes a policy-based IPsec VPN policy.
+    required: false
+    choices: ["deny", "accept", "ipsec"]
+
+  vpn_dst_node:
+    description:
+      - EXPERTS ONLY! KNOWLEDGE OF FMGR JSON API IS REQUIRED!
+      - List of multiple child objects to be added. Expects a list of dictionaries.
+      - Dictionaries must use FortiManager API parameters, not the ansible ones listed below.
+      - If submitted, all other prefixed sub-parameters ARE IGNORED. This object is MUTUALLY EXCLUSIVE with its options.
+      - We expect that you know what you are doing with these list parameters, and are leveraging the JSON API Guide.
+      - WHEN IN DOUBT, OMIT THE USE OF THIS PARAMETER AND USE THE SUB OPTIONS BELOW INSTEAD TO CREATE OBJECTS WITH MULTIPLE TASKS
+    required: false
+
+  vpn_dst_node_host:
+    description:
+      - NO DESCRIPTION PARSED ENTER MANUALLY
```

Comment:
Starting here, there are a few of these 'NO DESCRIPTION PARSED...' descriptions. I'm guessing you need to manually add something to each of this to describe the parameters?

Reply:
Yep! Sorry I'm adding a #TODO tag in this workflow so we don't miss it in the future.